### DISCUSSION TITLE
Table 2 Members

### DISCUSSION THREAD

**Discussion by rachael-clark** [IBAHCM/RPiR#411] [Old]
Rachael Clark: rachael-clark : https://github.com/IBAHCM/rachael-clark_series01.git

**Comment by emmamclea** [comment]
Emma McLea: emmamclea : https://github.com/IBAHCM/emmamclea_0201.git

**Comment by NinaGangi** [comment]
Nina Gangi: NinaGangi: https://github.com/IBAHCM/NinaGangi_series01.git

**Comment by nadiareeve** [comment]
Nadia Reeve: nadiareeve: https://github.com/IBAHCM/nadiareeve_0106.git

**Comment by emmamclea** [comment]
Hi guys, I've opened issues for each practical within my repository if you would be able to go and check them out for me! 
My link is:  https://github.com/IBAHCM/emmamclea_series02.git
